src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -0,0 +1,86 @@
+const BinarySocket     = require('../../../../base/socket');
+const localize         = require('../../../../../_common/localize').localize;
+const Url              = require('../../../../../_common/url');
+
+const AccountClosure = (() => {
+    const form_selector = '#form_closure';
+
+    const onLoad = () => {
+        $(form_selector).on('submit', (event) => {
```

Comment:
any reason we're not using our form manager for submission and error handling?

Reply:
Because of the validation. There wasn't a conditional validation for radio button (if this radio button is selected, validate the textfield this way)

Reply:
I see